virtlet_deploy/compose-and-upload-unikernels.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+VIRTLET_POD="${1:-}"
+PROJECT_DIR=$HOME/git-repos/osv-microservice-demo
```

Comment:
Can this be relative to your current path? It will otherwise have to be modified by users.